.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
```

Comment:
maybe we can change for bun or at least pnpm here?

Reply:
is there a way to inherit the package-ecosystem? because core at this moment, utilizes npm. the rc/bun tests, uses bun. the cli uses pnpm for the convenience of monorepo.

Reply:
I quite didn't understand what you mean by inherit package-ecosystem, you can change this file by branch but I don't know if dependabot by default will recognize it (or if he watches all branches by default or only "main" one), cli is another project so I don't hink that applies to this case.

We can still use NPM for now, and when rc/bun tests goes live we just adjust the line. @devmozao If it's ok and clarified for you I think we can apply the merge :)